navoica_enroll/static/js/project.js

```diff
@@ -260,7 +260,7 @@ function loadRegisterForm() {
       },
     },
     postal_code: {
-      pattern: /^[a-z0-9][a-z0-9\-]{0,10}[a-z0-9]\S+$/,
+      pattern: /^[a-z0-9][a-z0-9\- ]{0,10}[a-z0-9]\S+$/i,
```

Comment:
Nie wychwyciłam tego wcześniej, ale wydaje mi się, że bez \S+ na końcu, bo to wymusza jeszcze dodatkowy znak/znaki i pozwala żeby to była na przykład kropka. Nie waliduje też dwucyfrowych kodów poczowych.
Na regextesterze sprawdzałam /^[a-z0-9][a-z0-9\- ]{0,10}[a-z0-9]$/i  i wydaje mi się, że odpowiada naszym potrzebom i jest zgodne z rozwiązaniem ze https://stackoverflow.com/questions/578406/what-is-the-ultimate-postal-code-and-zip-regex odpowiedź Neil McGuigan

Reply:
/S odpowiada za niemożliwość wstawienia pustych spacji, ale + na końcu komplikował sprawę w przypadku dwycyfrowych kodów pocztowych. Zmieniłam to.